tests/basics/unary_op_builtin.py

```diff
@@ -0,0 +1,7 @@
+# check user instances derived from builtin without unary op handling for bool.
+class A(type):
```

Comment:
I think this test should just go in the existing `unary_op.py` file.  There's no longer a reason it needs to be separated (it should be able to run on all uPy ports).

Reply:
Yep, good point. Updated.